packages/@pollyjs/adapter/src/utils/stringify-request.js

```diff
@@ -1,4 +1,12 @@
 export default function stringifyRequest(req, ...args) {
+  const config = { ...req.config };
+
+  // Remove all adapter & persister config options as they can cause a circular
+  // structure to the final JSON
+  ['adapter', 'adapterOptions', 'persister', 'persisterOptions'].forEach(
```

Comment:
Is this a fix for separate bug?

Reply:
This is a fix for a separate bug that I found when tackling this.